### DISCUSSION TITLE
OCR - Empty spaces between characters

### DISCUSSION THREAD

**Discussion by AcidReign81** [jonaswinkler/paperless-ng#573] [Support]
Hi, 

I am pretty new to this project but I am loving it allready. 
I setup everything and it works like a charm.
I only have one downside at the moment and thats the OCR.
It gets most of the word and characters fine but in a lot of times it adds empty spaces. For example "Invoice" in the ocr is changed to "I n v o i c e".
Because of that all my automatic tags do not work.
Any ideas what I can do or tweak?

Thank you in advance

**Comment by jonaswinkler** [comment]
Hello! Are you running the docker container?

**Reply by AcidReign81** [reply]
yes. I am running the docker on unRaid

**Reply by jonaswinkler** [reply]
I'd wager that this is due to the version of `tesseract-ocr` (this is the library that performs OCR and is ultimately responsible for the output) being somewhat out of date. However, I can't update that right now, updating tesseract will break compatibility with ARM devices.

If you could upload a demo file on here that does not contain any private information, that would be great.

**Reply by AcidReign81** [reply]
Actually I think it was an error done by my scanner. The OCR was done in beforehand and then paperless didnt do the ocr again. Now everything seems so be fine. Thank you
